docs/reference/contrib/simple_translation.rst

```diff
@@ -0,0 +1,31 @@
+.. _simple_translation:
+
+Simple translation
+==================
+
+Simple Translation is a user interface that allows to create copies of translatables (pages and snippets) into another language.
```

Comment:
Maybe start this off like "The ``simple_translation`` module provides a user interface that allows users to copy pages and translatable snippets into another language"?